README.md

```diff
@@ -64,11 +64,12 @@ make llvm-spirv -j`nproc`
 
 ## Test instructions
 
-All tests related to the translator are placed in the [test](test) directory.
+All tests related to the translator are placed in the [test](test) directory. The tests require a recent enough SPIRV-Tools installation.
```

Comment:
This text needs to be updated to say that SPIRV-Tools are optional, not required.